src/common/motion/position.ts

```diff
@@ -410,13 +410,17 @@ export class Position extends vscode.Position {
     }
   }
 
-  public getRightThroughLineBreaks(): Position {
+  public getRightThroughLineBreaks(includeEol = false): Position {
```

Comment:
Note that this flag has the opposite default value from `getLeftThroughLineBreaks`, which is a bit confusing.
I erred on the side of changing fewer lines of code here, over making it more consistent.